src/main.tsx

```diff
@@ -13,22 +13,25 @@ import ProductShop from "./pages/product-shop/product-shop.tsx";
 import { apiGetCategories, CategoriesModel } from "./api/api-get-categories.ts";
 
 export async function loader() {
-  const category : CategoriesModel[] | undefined = await apiGetCategories("https://lopi2-backend-5517f8f04d28.herokuapp.com/api/");
-  return category ;
-} 
+  const category: CategoriesModel[] | undefined = await apiGetCategories(
+    "https://lopi2-backend-5517f8f04d28.herokuapp.com/api/"
```

Comment:
Please move all references to the backend to an environment variable (https://lopi2-backend-5517f8f04d28.herokuapp.com).